tests/graph/test_bellman_ford.py

```diff
@@ -175,7 +175,7 @@ def test_raises_negative_cycle_bellman_ford_paths(self):
         with self.assertRaises(rustworkx.NegativeCycle):
             rustworkx.bellman_ford_shortest_paths(graph, 0, weight_fn=float)
 
-    def test_raises_negative_cycle_bellman_ford_path_lenghts(self):
+    def test_raises_negative_cycle_bellman_ford_path_lengths(self):
```

Comment:
I made this typo, it is silly but this is one of the cases where the tool blocking CI would not add much value. The test is still correct.